docs/tutorials/custom/custom-base-dog.md

```diff
@@ -39,14 +36,14 @@ You’ll learn to implement a custom component type in Viam, and you’ll be equ
 
 ## Code used in this tutorial
 
+- [Module Generator](https://github.com/viam-labs/generator-viam-module)
 - [Robot Dog Base Code on GitHub](https://github.com/viam-labs/robot-dog-base)

```

Comment:
Shouldn't this code be updated to match the layout created by the module generator?

Also if the following can't point to 127.0.0.1, make it configurable via config attributes?
        self.client_socket.connect(("PASTE DOG IP ADDRESS HERE", 5001))

Reply:
Yeah--it is already configurable via config attributes in robotdog.py, but I will make an entirely new repo and link to that